### PR TITLE
[WIP] fix: cypress dashboard edit mode test

### PR DESCRIPTION
**Pull request by eschutho**
### SUMMARY
Fixes broken/flaky test
### TEST PLAN
should pass on cypress
### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/cypress-base/cypress/integration/dashboard/edit_mode.test.js [hunk 1]
```diff
@@ -23,34 +23,35 @@ describe('Dashboard edit mode', () => {
...
 
   it('remove, and add chart flow', () => {
-    // wait for box plot to appear
-    cy.get('.grid-container .box_plot');
-
-    cy.get('.fa.fa-trash')
-      .last()
+    // wait for world map to appear
+    cy.wait('@worldMap');
+    cy.get('.world_map', { timeout: 60000 })
```
Comment (ktmud) [inline superset-frontend/cypress-base/cypress/integration/dashboard/edit_mode.test.js:33]: Why wait 1 minute? If I'm understanding Cypress right, after you waited for the API requests, the additional wait time is only about UI render time. No matter how long you wait, the element may still be undiscoverable to Cypress if something is broken.

Comment (graceguo-supercat) [reply superset-frontend/cypress-base/cypress/integration/dashboard/edit_mode.test.js:33]: I do not think the flaky problem is caused by `box_plot` chart

Comment (eschutho) [reply superset-frontend/cypress-base/cypress/integration/dashboard/edit_mode.test.js:33]: @graceguo-supercat I agree, I don't think the `box_plot` is the issue. Apologies for the confusion, I posted this to test the ci while working out CSRF on Cypress and docker together locally. Will update this test here in a bit.

**Comment by eschutho** [comment]
@adam-stasiak is going to help out with Cypress tests, so closing this PR.

**Pull request closed without merging**